README.mkd

```diff
@@ -59,6 +59,24 @@ cd ~/.vim/bundle/vimproc.vim
 make
 ```
 
+### dein.vim
+
+You can leverage dein.vim to install and manage vimproc.vim.
+
+```
+call dein#add('Shougo/vimproc.vim', {
+\ 'build' : {
+\     'windows' : 'tools\\update-dll-mingw',
+\     'cygwin' : 'make -f make_cygwin.mak',
+\     'mac' : 'make',
+\     'linux' : 'make',
+\     'unix' : 'gmake',
+\    },
```

Comment:
This is wrong.  You cannot pass the dictionary to the "build".
Please use `'build': 'make'`